app_legacy/Helpers/render/layout.php

```diff
@@ -127,18 +141,18 @@ function RenderToolbar(): void
     echo "<li><a href='#'>Community</a>";
     echo "<div>";
     echo "<ul>";
-    echo "<li><a href='/forum.php'>Forums</a></li>";
-    echo "<li><a href='/forum.php?c=1'>- Community</a></li>";
-    echo "<li><a href='/viewforum.php?f=25'>+- Competitions</a></li>";
-    echo "<li><a href='/forum.php?c=7'>- Developers</a></li>";
-    echo "<li><a href='/forumposthistory.php'>Recent Posts</a></li>";
+    echo "<li><a href='https://news.retroachievements.org/'>RANews</li>";
+    echo "<li><a href='https://github.com/RetroAchievements/guides/wiki/'>RAGuides Wiki</li>";
```

Comment:
I really think the forums should be the first items in the list and these should be moved down towards the documentation. I'd probably change the order of some of the other items too:
```
Forum Index
Event Forums
Recent Forum Posts
-----
User List
Global Ranking
Recent Masteries
Developer Stats
-----
RANews
RAGuides Wiki
-----
User Documentation
Developer Documentation
```

Reply:
Resorted per the list